### PR TITLE
feat: add port rest template

### PR DESCRIPTION
**Pull request by rahulguptajss**

### CODE REVIEW

File: conf/rest/9.13.0/snapshot.yaml [hunk 1]
```diff
@@ -2,10 +2,6 @@ name:                     Snapshot
 query:                    api/private/cli/snapshot/policy
 object:                   snapshot
 
-schedule:
```
Comment (cgrinds) [inline conf/rest/9.13.0/snapshot.yaml:5]: Unrelated change?

**Comment by cgrinds** [comment]
Looks good, one minor comment

**Review by cgrinds — APPROVED** [review]

File: conf/rest/9.13.0/snapshot.yaml (hunk 1 above)
Comment (rahulguptajss) [reply conf/rest/9.13.0/snapshot.yaml:5]: yes added in this PR based on hardik's changes in https://github.com/NetApp/harvest/pull/778
